.github/workflows/ci.yml

```diff
@@ -47,13 +45,11 @@ jobs:
         TARGET: [x86_64-unknown-linux-gnu, thumbv6m-none-eabi, thumbv7m-none-eabi]
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
Can we pin tool chain to a version? I don't like running off master in case it causes random breakage personally

Reply:
dtolnay implemented it such that the version tags correspond to the toolchain version.  `master` is used when the toolchain version is parametric.

I'm not too concerned about stability since dtolnay has a good record of API stability from serde and proc macros, and I have not had any issues in ~1yr of using it for my own repos.

That all said, I'm happy to find something else if you disagree :)